ui/react-components/api/appointmentsApi.js

```diff
@@ -10,3 +10,10 @@ export const saveOrUpdateAppointment = async (data) => {
     }
 };
 
+export const getConflicts = async appointmentRequest => {
```

Comment:
can we call it conflictsFor instead of getConflicts?

Reply:
changed